help/access/access.md

```diff
@@ -1,24 +1,110 @@
+below:
+
+| Data source          | Query      |
+|----------------------|------------|
+| Relational databases | SQL        |
+| File share           | Filename   |
+| Excel file           | Sheet name |
+| Linked data          | SPARQL     |
+| Box                  | Filename   |
+
+The result of a query execution is represented by the [function call](../datagrok/functions/function-call.md). For instructions on how to run, share queries, and manage queries prorammatically, see [developer's documentation](../develop/how-to/access-data.md).
+
+A data query is an [entity](../datagrok/objects.md), which means it can be shared, assigned permissions, annotated, and more.
+
+## Data formats
+
+With Datagrok, you can retrieve both structured and unstructured data. Datagrok supports [multiple data formats](supported-formats.md), including popular formats like CSV, TXT, JSON, and scientific formats like MAT, molecular structure formats (like PDB, MOL, or SDF), geographic annotation, and others. Datagrok also offers a flexible system for extending the platform with organization-specific data formats (see [Extensible framework](#extensible-framework)).
+
+## Browsing and preview
+
+Datagrok offers an array of capabilities and features designed to help users efficiently browse, manage, and preview the content of their data. For more information, see:
+
+* [Database Manager](databases.md/#database-manager)
+* [File Manager](file-shares.md/#file-manager)
+* [Webservices Manager](open-api.md/#webservices-manager).
+
+## Sharing and access control
+
+Datagrok treats data connections, file shares, database tables and columns, and queries as [entities](../datagrok/objects.md), which means there is a comon set of operations that can be applied to them. These entities can be shared with others, assigned access privileges, commented on, versioned, [audited](../govern/audit.md), and so on. Some of the most popular privileges are: `view`, `edit`, `delete`, and `share`. These privileges can be given to individual users, or
+to [groups](../govern/group.md). For more information on the access privilege model, see [Privileges](../govern/security.md#privileges).
+
+Data connections can be shared as part of a [project](../datagrok/project.md), [package](../develop/develop.md#packages) (and [repository](connectors/git.md) containing this package), or as a standalone [entity](../datagrok/objects.md). The access rights of a database connection are inherited from the access rights of a query. However, the reverse is not true: access rights of a query don't inherit the access rights of the database connection. Consequently, when sharing a query, the associated database connection is shared automatically. On the other hand, sharing a database connection does not automatically share your queries. For web queries, they are shared automatically when the corresponding connection is shared.
```

Comment:
This is a bit unclear. 

> The access rights of a database connection are inherited from the access rights of a query.
I think it's the other way around?

Probably not "Consequently", but "To minimize the confusion" or "To make queries work when a query is shared" or something like this.